CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## Unreleased
 
 ## v0.20.4
-### April 29, 2025
```

Comment:
I assume there was no release on April 29, we just pre-prepared the changelog without releasing then?

Reply:
Yep!